packages/lesswrong/components/notifications/NotificationsMenu.jsx

```diff
@@ -4,11 +4,11 @@ import PropTypes from 'prop-types';
 import Drawer from 'material-ui/Drawer';
 import Badge from 'material-ui/Badge';
 import {Tabs, Tab} from 'material-ui/Tabs';
-import AllIcon from '@material-ui/icons/Notifications';
-import ClearIcon from '@material-ui/icons/Clear';
-import PostsIcon from '@material-ui/icons/Description';
-import CommentsIcon from '@material-ui/icons/ModeComment';
-import MessagesIcon from '@material-ui/icons/Forum';
+import AllIcon from 'material-ui/svg-icons/social/notifications';
+import ClearIcon from 'material-ui/svg-icons/content/clear';
+import PostsIcon from 'material-ui/svg-icons/action/description';
+import CommentsIcon from 'material-ui/svg-icons/editor/mode-comment';
+import MessagesIcon from 'material-ui/svg-icons/communication/forum';
```

Comment:
Why did you revert the icons to the old material-ui format here? Did anything break?

Reply:
Ugh. I didn't - this is some sort of merge artifact. I'll investigate further when I get the chance tonight; best guess is I did a git operation with the file open in editor with unsaved changes. Usually I look at diffs as I'm adding things to a commit, but I must not have this time.